### ISSUE TITLE
[feature request]: ChatGPTActAs allows user to edit prompt before submitting

### ISSUE THREAD

**Issue by rensftw** [jackMort/ChatGPT.nvim#182]
`ChatGPTActAs` is a very powerful command and it would be even better if the user could edit the Awesome Prompt text before the request gets submitted.

**Current behaviour**
`ChatGPTActAs` shows a preview and title for awesome prompts, but does not allow prompt to be yanked.
I have tried `<C-w w>`, but this causes the prompt to close instead of letting me navigate to the prompt preview window and yank the prompt (see video below 👇) .

https://user-images.githubusercontent.com/22574186/236442926-9d1f104a-c2e2-4312-904f-d822affadc32.mp4

Once you choose a prompt it is immediately submitted to the OpenAI endpoint and consumes tokens, even if the prompt is not customized/ready to be submitted yet.

**Proposed behaviour**
`ChatGPTActAs` shows a preview and title for awesome prompts; hitting enter (or some other keybinding) opens a new session and populates the `prompt` field so it can be modified by the user before submitting. 
<img width="1496" alt="image" src="https://user-images.githubusercontent.com/22574186/236444545-b81fff38-609c-4531-be29-3076f1929bd6.png">

PS: Thank you for the great plugin! ❤️ It's my favourite way of interfacing with ChatGPT

**Comment by jackMort** [comment]
I added option to set "system message" it should solve the issue, need to update the README  later to cover that.

**Comment by rensftw** [comment]
Thanks for the fast response ⚡ 

I tested it out and the new `SYSTEM` window looks really good! My first instinct was to `jk` for moving but I realised that the prompt is one single wrapped line.

https://user-images.githubusercontent.com/22574186/236673735-d14d1e5b-f58a-48c5-ba35-aad4447668bb.mp4

Btw, how should `draft_message = "<C-d>"` be used? 
In `ChatGPTActAs` it just scrolls the preview buffer (usual Vim movement)
In the regular prompt it seems to send an empty message and nothing else?

https://user-images.githubusercontent.com/22574186/236674040-f81995cb-4eba-41a2-9965-a63330037dec.mp4
